src/Nethermind/Chains/mekong.json

```diff
@@ -74,6 +74,12 @@
     "extraData": "",
     "gasLimit": "0x17d7840"
   },
+  "nodes":
```

Comment:
Already has empty `nodes` item at bottom, so that would overwrite this one being later?

Reply:
good catch

Reply:
Actually started before noticing that and bootnodes were not loaded correctly :) So interesting thing for future.